pkgs/top-level/all-packages.nix

```diff
@@ -32897,6 +32897,8 @@ with pkgs;
 
   opentx = libsForQt5.callPackage ../applications/misc/opentx { };
 
+  openvi = callPackage ../applications/editors/openvi { };
```

Comment:
```suggestion
  openvi = darwin.apple_sdk_11_0.callPackage ../applications/editors/openvi { };
```

Reply:
What about Linux?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/platform-notes.chapter.md

> On Linux, this will have the same effect as `pkgs.callPackage`, so you can use `pkgs.darwin.apple_sdk_11_0.callPackage` regardless of platform.

Reply:
But I don't own any Mac device, so I can't test if it builds or not, nor can I update the apple sdk when needed.

Reply:
In this case, you can let ofborg do the work, or just tag any darwin maintainer.